src/components/structures/RoomView.tsx

```diff
@@ -1208,12 +1211,30 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
             case Action.EditEvent: {
                 // Quit early if we're trying to edit events in wrong rendering context
                 if (payload.timelineRenderingType !== this.state.timelineRenderingType) return;
+                if (payload.event && payload.event.getRoomId() !== this.state.roomId) {
+                    // If the event is in a different room, we need to view that room first
```

Comment:
how on earth would we get an "editevent" for an event in another room?

Reply:
Room > Search > All rooms - shows a timeline of events from all rooms